packages/nova-react/README.md

```diff
@@ -197,3 +197,135 @@ const MyComponentWrapper = () => {
 
 You can nest as many interceptors as you need to either handle or pass events further up.
+
+## Localization
+
+> Note for this to work you will need to use the Microsoft internal fork of the Relay compiler that includes the added functionality.
+
+Localized strings are provided to components through GraphQL fragments. For simple strings these will be typed as `string` and can be used directly. For strings that include placeholders Relay the compiler will type them as `StringWithPlaceholders` from `@nova/react`.
+
+This is an opaque type that includes the placeholders required by the string. You can use the `format` function provided by `useFormat` from `@nova/react` to format the string with the required values.
+
+### Defining a localized string
+
+You can define a localized string in your GraphQL schema by using the `@localizedString` directive. This directive takes a `text` argument which is the string with placeholders and a `comment` argument which is a description of the string.
+
+#### Simple string
+
+```graphql
+type ViewData {
+  greeting: String! @localizedString(
+    text: "Welcome!",
+    comment: "Greeting to the user",
+  )
+}
+```
+
+In typescript this will be typed as `string`.
+
+```ts
+type ViewData = {
+  greeting: string;
+};
+```
+
+#### String with placeholders
+
+In the case of a string with placeholders you will also need to provide a `placeholders` argument which is a list of objects with a `name` and `comment` field.
+
+```graphql
+type ViewData {
+  greeting: String! @localizedString(
+    text: "Hello, {name}",
+    comment: "Greeting to the user",
+    placeholders: [
+      {
+        name: "name",
+        comment: "The name of the user"
+      }
+    ]
+  )
+}
+```
+
+In typescript this will be typed as `StringWithPlaceholders`.
+
+```ts
+type ViewData = {
+  greeting: StringWithPlaceholders<{ name: string; }>;
+};
+```
+
+### Consuming a localized string
+
+When consuming a localized string you can use it directly if it is a simple string.
+
+```tsx
+import { useFragment } from "@nova/react";
+
+const MyComponent = ({ viewData }) => {
+  const { greeting } = useFragment(
+    graphql`
+      fragment MyComponent_viewData on ViewData {
+        greeting
+      }
+    `
+    viewData
+  )
+
+  return <div>{greeting}</div>;
+};
+```
+
+### Consuming a localized string with placeholders
+
+When consuming a localized string with placeholders you will need to use the `format` function provided by `useFormat` from `@nova/react`. This function takes the string and an object with the values for the placeholders.
+
+```tsx
+import { useFragment, useFormat } from "@nova/react";
+
+const MyComponent = ({ viewData }) => {
+  const { greeting } = useFragment(
+    graphql`
+      fragment MyComponent_viewData on ViewData {
+        greeting
+      }
+    `
+    viewData
+  )
+
+  const format = useFormat();
+
+  return <div>{format(greeting, { name })}</div>;
+};
+```
+
+### Providing the `useFormat` hook implementation
```

Comment:
Apart from update in readme, could we also update `Feedback` in `examples` folder to use localization in some way?

Reply:
We could add localization without placeholders but for any strings with placeholders we'll need compiler support.

Reply:
Done for both and reworked the typings a bit to allow it without type safe placeholders.